tests/unit/store/gateway/mutations.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { mutations } from '@/store/modules/gateway/mutations';
+import { state } from '@/store/modules/gateway/state';
+import * as mutation from '@/store/modules/gateway/types-mutations';
+import * as types from '@/store/types';
+
+describe(mutation.SET_AUTH_HEADER, () => {
+  const { SET_HTTP_HEADER } = mutations;
+
+  it('should equal to payload', () => {
+    const hostname: string = 'test-address';
+    SET_HTTP_HEADER(state, hostname);
+    expect(state.authHeader).toEqual(hostname);
+  });
+});
+
+// https://github.com/grpc/grpc-web/pull/540
```

Comment:
We need a task to track this.

Reply:
https://github.com/hwsc-org/hwsc-frontend/issues/27